Add UnnamedDiscriminatedUnionFieldAnalyzer (#10)
ionide/ionide-analyzers: docs/suggestion/004.md

---
title: UnnamedDiscriminatedUnionField
category: suggestion
categoryindex: 2
index: 3
---
# UnnamedDiscriminatedUnionFieldAnalyzer

## Problem

In [dotnet/fsharp#15665](https://github.com/dotnet/fsharp/issues/15665) it is recommended that all union case fields have a name.  

> With the exception that for a single field union case.

Reviewer: ```suggestion
With the exception of a single field union case.
```
